interface/build.config.js

```diff
@@ -79,8 +79,13 @@ module.exports = {
       'vendor/angularjs-slider/dist/rzslider.min.js',
       'vendor/d3-tip/index.js',
       'node_modules/hclusterjs/hcluster.js',
+
+      // note: greenelab.stats.ttest.js and greenelab.stats.multtest.js are
+      // generated via browserify. greenelab.stats.ttest.js also needs to use
+      // babel to be able to be browserified.
```

Comment:
To be precise, browserifying works fine without the babel step, but we need to use babel for compatibility. The `ttest` code was written in ES6 and all of our other JavaScript is in ES5. Beyond simply standardizing on ES5, we use babel so `ttest`s will run without issues on a broad spectrum of browsers.

Side note: in addition, our `uglify` minification step cannot handle certain ES6 constructs yet. Work is being done on that, but the future of minification might actually turn out to be using babel...